composer.json

```diff
@@ -23,7 +23,10 @@
         "matthiasnoback/symfony-dependency-injection-test": "~0.6",
         "matthiasnoback/symfony-config-test": "^1.3.1",
         "phpunit/php-code-coverage": "@stable",
-        "sonata-project/doctrine-phpcr-admin-bundle": "^1.1",
+        "sonata-project/admin-bundle": ">=2.2.12",
+        "sonata-project/core-bundle": ">=2.2.3",
```

Comment:
the caret operator means this patch version or any minor version higher than this.